docs/machine-learning/how-to-choose-an-ml-net-algorithm.md

```diff
@@ -54,50 +91,144 @@ Decision tree algorithms take more resources and do not scale as well as linear
-|Fast forest|Works well with noisy data|<xref:Microsoft.ML.Trainers.FastTree.FastForestBinaryTrainer> <xref:Microsoft.ML.Trainers.FastTree.FastForestRegressionTrainer>|
-|Generalized additive model (GAM)|Best for problems that perform well with tree algorithms but where explainability is a priority|<xref:Microsoft.ML.Trainers.FastTree.GamBinaryTrainer> <xref:Microsoft.ML.Trainers.FastTree.GamRegressionTrainer>|
+Fastest and most accurate of the binary classification tree trainers. Highly tunable.
+
+|Trainer|Task|ONNX Exportable|
+|---------|----------|----------|
+|<xref:Microsoft.ML.Trainers.LightGbm.LightGbmBinaryTrainer>|Binary classification|Yes|
+|<xref:Microsoft.ML.Trainers.LightGbm.LightGbmMulticlassTrainer>|Multiclass classification|Yes|
+|<xref:Microsoft.ML.Trainers.LightGbm.LightGbmRegressionTrainer>|Regression|Yes|
+|<xref:Microsoft.ML.Trainers.LightGbm.LightGbmRankingTrainer>|Ranking|No|
+
+### Fast tree
+
+Use for featurized image data. Resilient to unbalanced data. Highly tunable.
+
+|Trainer|Task|ONNX Exportable|
+|---------|----------|----------|
+|<xref:Microsoft.ML.Trainers.FastTree.FastTreeBinaryTrainer>|Binary classification|Yes|
+|<xref:Microsoft.ML.Trainers.FastTree.FastTreeRegressionTrainer>|Regression|Yes|
+|<xref:Microsoft.ML.Trainers.FastTree.FastTreeTweedieTrainer>|Regression|Yes|
+|<xref:Microsoft.ML.Trainers.FastTree.FastTreeRankingTrainer>|Ranking|No|
+
+### Fast forest
+
+Works well with noisy data.
+
+|Trainer|Task|ONNX Exportable|
+|---------|----------|----------|
+|<xref:Microsoft.ML.Trainers.FastTree.FastForestBinaryTrainer>|Binary classification|Yes|
+|<xref:Microsoft.ML.Trainers.FastTree.FastForestRegressionTrainer>|Regression|Yes|
+
+### Generalized additive model (GAM)
+
+Best for problems that perform well with tree algorithms but where explainability is a priority.
+
+|Trainer|Task|ONNX Exportable|
+|---------|----------|----------|
+|<xref:Microsoft.ML.Trainers.FastTree.GamBinaryTrainer>|Binary classification|No|
+|<xref:Microsoft.ML.Trainers.FastTree.GamRegressionTrainer>|Regression|No|
 
 ## Matrix factorization
 
-|Properties|Trainers|
-|----------|--------|
-|Best for sparse categorical data, with large datasets|<xref:Microsoft.ML.Trainers.FieldAwareFactorizationMachineTrainer>|
+### Matrix Factorization
+
+Used for [collaborative filtering](https://en.wikipedia.org/wiki/Collaborative_filtering) in recommendation.
+
+|Trainer|Task|ONNX Exportable|
+|---------|----------|----------|
+|<xref:Microsoft.ML.Trainers.MatrixFactorizationTrainer>|Recommendation|No|
+
+### Field Aware Factorization Machine
+
+ Best for sparse categorical data, with large datasets.
+
+|Trainer|Task|ONNX Exportable|
+|---------|----------|----------|
+|<xref:Microsoft.ML.Trainers.FieldAwareFactorizationMachineTrainer>|Binary classification|No|
 
 ## Meta algorithms
```

Comment:
@briacht are the meta algos also going into a table?

Reply:
Which ones? Ova and Pairwise are already in tables